CONTRIBUTING.rst

```diff
@@ -151,8 +151,8 @@ but here is a simple overview:
 
     $ git clone git@github.com:YOUR_GITHUB_USERNAME/pytest.git
     $ cd pytest
-    $ git checkout pytest-2.7      # if you want to fix a bug for the pytest-2.7 series
-    $ git checkout master          # if you want to add a feature bound for the next minor release
+    $ git checkout 2.8.X   # if you want to fix a bug for the pytest-2.8 series
```

Comment:
right, i forgot about this.  I am open to trying this out master for bugfixes and maybe future-2.9 to make it clear that 2.9 does not exist yet (i am fan of a bit of redundancy)